source/main/gui/panels/GUI_NodeBeamUtils.cpp

```diff
@@ -151,20 +153,20 @@ void NodeBeamUtils::Draw()
     ImGui::Spacing();
     if (actor->ar_nb_initialized)
     {
-        ImGui::Columns(2, "Search results");
+        ImGui::Columns(2, _L("Search results"));
         ImGui::SetColumnOffset(1, 290.f);
-        ImGui::Text("Reference");
+        ImGui::Text(_L("Reference"));
         ImGui::NextColumn();
-        ImGui::Text("Optimum");
+        ImGui::Text(_L("Optimum"));
         ImGui::NextColumn();
         ImGui::Separator();
-        ImGui::Text("Movement: %f (%f)",   actor->ar_nb_reference[5] / actor->ar_num_nodes, actor->ar_nb_reference[4]);
-        ImGui::Text("Stress: %.2f (%.2f)", actor->ar_nb_reference[1] / actor->ar_num_beams, actor->ar_nb_reference[0]);
-        ImGui::Text("Yitter:   %f (%f)",   actor->ar_nb_reference[3] / actor->ar_num_beams, actor->ar_nb_reference[2]);
+        ImGui::Text(_L("Movement: %f (%f)"),   actor->ar_nb_reference[5] / actor->ar_num_nodes, actor->ar_nb_reference[4]);
+        ImGui::Text(_L("Stress: %.2f (%.2f)"), actor->ar_nb_reference[1] / actor->ar_num_beams, actor->ar_nb_reference[0]);
+        ImGui::Text(_L("Yitter:   %f (%f)"),   actor->ar_nb_reference[3] / actor->ar_num_beams, actor->ar_nb_reference[2]);
         ImGui::NextColumn();
-        ImGui::Text("Movement: %f (%f)",   actor->ar_nb_optimum[5] / actor->ar_num_nodes, actor->ar_nb_optimum[4]);
-        ImGui::Text("Stress: %.2f (%.2f)", actor->ar_nb_optimum[1] / actor->ar_num_beams, actor->ar_nb_optimum[0]);
-        ImGui::Text("Yitter:   %f (%f)",   actor->ar_nb_optimum[3] / actor->ar_num_beams, actor->ar_nb_optimum[2]);
+        ImGui::Text(_L("Movement: %f (%f)"),   actor->ar_nb_optimum[5] / actor->ar_num_nodes, actor->ar_nb_optimum[4]);
+        ImGui::Text(_L("Stress: %.2f (%.2f)"), actor->ar_nb_optimum[1] / actor->ar_num_beams, actor->ar_nb_optimum[0]);
+        ImGui::Text(_L("Yitter:   %f (%f)"),   actor->ar_nb_optimum[3] / actor->ar_num_beams, actor->ar_nb_optimum[2]);
```

Comment:
again... the language string shouldn't contain `printf()` format specifiers.

In this instance, I suggest doing 
```
ImGui::Text(_L("%s:   %f (%f)"), _L("ValueType"),   val1, val2);
```